applets/wncklet/wayland-backend.c

```diff
@@ -361,6 +361,7 @@ void
 wayland_tasklist_set_orientation (GtkWidget* tasklist_widget, GtkOrientation orient)
 {
 	TasklistManager *tasklist = tasklist_widget_get_tasklist (tasklist_widget);
+	g_return_if_fail(tasklist);
```

Comment:
A space is missing before opening bracket.
I mean this is your code-style in this file :)